NITF driver: implement write support for CADRG (Compressed ARC Digitized Raster Graphics) (#13940)
OSGeo/gdal: doc/source/drivers/raster/rpftoc.rst

.. supports_georeferencing::

.. supports_virtualio::

Open options
------------

|about-open-options|
This driver supports the following open options:

- .. oo:: FORCE_RGBA
     :choices: YES, NO
     :default: NO
     :since: 3.13

     In some situations, :ref:`raster.nitf` tiles inside a subdataset
     don't share the same palettes. The RPFTOC driver will do its best to
     remap palettes to the reported palette by gdalinfo (which is the palette

>      of the first tile of the subdataset). If it does not give a good result,

Reviewer: ```suggestion
     of the first tile of the subdataset). If it doesn't produce the desired result,
```

>      you can set this option to YES t get a RGBA dataset, instead of a paletted

Author: fixed
